video_creation/data/videos.json

```diff
@@ -1 +0,0 @@
-[]
```

Comment:
Can you revert this change? If I believe correctly, this will cause the bot to crash.